src/openrct2-ui/windows/EditorInventionsList.cpp

```diff
@@ -786,16 +788,17 @@ static void window_editor_inventions_list_drag_cursor(
  *
  *  rct2: 0x00685412
  */
-static void window_editor_inventions_list_drag_moved(rct_window* w, ScreenCoordsXY screenCoords)
+static void window_editor_inventions_list_drag_moved(rct_window* w, const ScreenCoordsXY& screenCoords)
 {
     ResearchItem* researchItem;
 
     int32_t scrollId;
     // Skip always researched items, so that the dragged item gets placed underneath them
+    auto newScreenCoords = screenCoords;
```

Comment:
Is there any advantage to doing this over leaving the signature non-const?

Reply:
Other than making interfaces consistent, I think there is not.